sdk/ml/azure-ai-ml/azure/ai/ml/entities/_notification/notification.py

```diff
@@ -0,0 +1,35 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+from typing import List, Optional
+
+
+from azure.ai.ml._restclient.v2023_02_01_preview.models import NotificationSetting as RestNotificationSetting
+from azure.ai.ml.entities._mixins import RestTranslatableMixin
+from azure.ai.ml._utils._experimental import experimental
+
+
+@experimental
+class Notification(RestTranslatableMixin):
```

Comment:
I think we should name Notifcation as "NotificationConfiguration" or "NotificationSettings"

Reply:
Is there already Notification class defined? This is a common concept, not featurestore specific.

Reply:
No, it is not in sdk yet. 
In api it is Notification setting